src/components/addTask.vue

```diff
@@ -3,48 +3,135 @@
     <h2>Add a New Task</h2>
     <form class="add-form">
       <label>Task Title:</label>
-      <input 
-        class="title-input" 
-        type="text" 
-        v-model="title" 
-        required />
+      <label 
+        class="title-input mdc-text-field mdc-text-field--filled mdc-text-field--no-label">
+        <span class="mdc-text-field__ripple"></span>
```

Comment:
This can be written as: `<span class="mdc-text-field__ripple" />` (Self Closing tag.) Update other areas as well.